### PR TITLE
Opportunity grant last call

### PR DESCRIPTION
**Pull request by LideviK**
Opportunity grant last call added, @tim-schilling @drewbrew

### CODE REVIEW

File: _posts/2024-04-22-opportunity-grant-last-call.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+uthor: DjangoCon US Communication Team
+category: General
+date: 2024-04-22 06:00:00
+layout: post
+image: /static/img/blog/cfp-oppgrants.jpg
```
Comment (tim-schilling) [inline _posts/2024-04-22-opportunity-grant-last-call.md]: I think this is the open graph image in which we'd want to have a text overlay.

File: _posts/2024-04-22-opportunity-grant-last-call.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+uthor: DjangoCon US Communication Team
```
Comment (tim-schilling) [inline _posts/2024-04-22-opportunity-grant-last-call.md]:
```suggestion
author: DjangoCon US Communication Team
```

File: _posts/2024-04-22-opportunity-grant-last-call.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+category: General
+date: 2024-04-22 06:00:00
+layout: post
+image: /static/img/blog/cfp-oppgrants.jpg
+post_photo_url: /static/img/blog/cfp-oppgrants.jpg
+post_photo_alt: "End of Our Opportunity Grants is coming up!"
+title: "Heads Up! End of Our Opportunity Grants is coming up!"
```
Comment (tim-schilling) [inline _posts/2024-04-22-opportunity-grant-last-call.md]:
```suggestion
post_photo_alt: "End of our Opportunity Grants is coming up!"
title: "Heads Up! End of our Opportunity Grants is coming up!"
```
I don't think Our should be capitalized.

File: _posts/2024-04-22-opportunity-grant-last-call.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+## Hey Django lovers, 
+
+here to remind you that the [opportunity grants](https://2024.djangocon.us/opportunity-grants/) will be closing on April 24, 2024 at 12 PM EDT.
```
Comment (tim-schilling) [inline _posts/2024-04-22-opportunity-grant-last-call.md]:
```suggestion
Here to remind you that the [opportunity grants](https://2024.djangocon.us/opportunity-grants/) will be closing on April 24, 2024 at [12 PM EDT](https://time.is/1200PM_24_Apr_2024_in_New_York?DjangoCon_US_2024_CFP_closes).
```

Let's use the timezone link from the OG page. And I think H should be capitalized here, similar to how a letter is written.

File: _posts/2024-04-22-opportunity-grant-last-call.md (hunk 1 above)
Comment (LideviK) [reply _posts/2024-04-22-opportunity-grant-last-call.md]: Added OG last call image.

**Review by jcjudkins — APPROVED** [review]
Looks Good!

**Review by tim-schilling — APPROVED** [review]

**Pull request merged**